release_notes.md

```diff
@@ -1,25 +1,30 @@
 # What's New
 
-With this update the new module **Threat Intel Reports** becomes a single pane of glass to analyze threat reports from FortiRecon ACI.
+**Changed default FortiGuard ingestion**:
 
-Indicators of Compromise (IOCs) in threat reports from FortiRecon ACI can be extracted as *Threat Intel Feeds* and correlated with threat reports for a holistic view of threats.
- 
-The scenario **Threat Intel Reports** now helps understand how *Threat Intel Management* solution pack collects reports and showcases correlations.
+- Changed the way feed records' description is formatted
 
-- A new module **Threat Intel Reports** now collects threat intel reports from FortiRecon ACI.
+- Changed default pull interval to 1 day to enhance performance.
+    <!-- - The last 7 days worth of feed is large and requires that the default request timeout values for `nginx` and `uwsgi` is tweaked for the first ingestion to complete. -->
 
-    - Threat intel reports module view has also been added.
+## Bug Fixes
 
-- A FortiRecon ACI connector now installs as part of the pre-requisites of this solution pack.
+- Fixed an issue where the Threat Intel Report's System View Template (SVT) did not display the report as designed.
 
-- Following new picklists have been added:
+- Trimmed the Threat Intel Report's detailed record to contain the FortiSOAR ID and not the fetched report's ID.
 
-    - Information Reliability
+## Known Issues
 
-    - Relevance Rating
-    
-    - Report Type
-    
-    - Source Category
-    
-    - Source Reliability
\ No newline at end of file
+- The following error message may appear during data ingestion:
+
+    ```console
+    Error: Process 1046904 waits for ShareLock on transaction 781491; blocked by process 1046907.
+    Process 1046907 waits for ShareLock on transaction 781489; blocked by process 1046904.
+    HINT:  See server log for query details.
+    CONTEXT:  while inserting index tuple (101319,1) in relation "threat_intel_feeds"
+    URL: https://localhost/api/ingest-feeds/threat_intel_feeds?$statusOnly=true
+    ```
+
+## Resolution
+
+Contact support.
```

Comment:
Please change this to Contact Fortinet support

Reply:
Fixed